Reply/app/src/main/java/com/materialstudies/reply/ui/home/EmailViewHolder.kt

```diff
@@ -44,7 +43,21 @@ class EmailViewHolder(
     init {
         binding.run {
             this.listener = listener
-            attachmentRecyclerView.adapter = attachmentAdapter
+            composeView.apply {
+                setViewCompositionStrategy(
+                    ViewCompositionStrategy.DisposeOnViewTreeLifecycleDestroyed
+                )
+                setContent {
+                    MdcTheme {
+                        EmailAttachmentRow(
+                            emailAttachments = email!!.attachments,
```

Comment:
Just confirming that it's safe to force unwrap here. Maybe add a comment if so?

Reply:
Decided to extract this entire section from `init` into the `bind` method